packages/atomic/src/components/common/interface/interface-common.tsx

```diff
@@ -2,18 +2,158 @@ import type {SearchEngine} from '@coveo/headless';
 import {LogLevel} from '@coveo/headless';
 import type {RecommendationEngine} from '@coveo/headless/recommendation';
 import type {InsightEngine} from '@coveo/headless/insight';
-import {i18n} from 'i18next';
+import {i18n, TFunction} from 'i18next';
+import {ComponentInterface, h} from '@stencil/core';
+import {setCoveoGlobal} from '../../../global/environment';
```

Comment:
I've merged the PR that introduces the aliases, so feel free to update `global`, `utils` and `@components/[common|search|recs]`